### PR TITLE
CVSL-2035 add cvl api endpoint to retrieve hdc licence data

### PR DESCRIPTION
**Pull request by louisenorris**

### CODE REVIEW

File: server/routes/viewingLicences/handlers/printLicence.test.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ describe('Route - print a licence', () => {
     qrCodeService.getQrCode = jest.fn()
     licenceService.getExclusionZoneImageData = jest.fn()
     licenceService.recordAuditEvent = jest.fn()
-    hdcService.getHdcInfo = jest.fn()
+    hdcService.getHdcLicenceData = jest.fn()
```
Comment (andrewrlee) [inline server/routes/viewingLicences/handlers/printLicence.test.ts:29]: **suggestions:(non-blocking)** we should probably turn these into proper jest mocks, I've been tidying them up as I hit them

**Review by andrewrlee — APPROVED** [review]
LGTM

File: server/views/pages/licence/HDC_AP.njk [hunk 2]
```diff
@@ -59,30 +59,39 @@
...
-      <li>Your supervision commences on <span class="entry">{{ licence.homeDetentionCurfewApprovedDate | datetimeToDate or "MISSING - HDC Approved Date"}}</span> and expires on <span class="entry">{{ licence.licenceExpiryDate | datetimeToDate or "MISSING - Licence expiry date" }}</span> unless this licence is previously revoked.</li>
+      <li>Your Home Detention Curfew commences on <span class="entry">{{ licence.actualReleaseDate | datetimeToDate or "MISSING - HDC Approved Date/Actual Release Date"}}</span> and expires on <span class="entry">{{ licence.conditionalReleaseDate | datetimeToDate or "MISSING - Conditional Release Date"}}</span></li>
+      <li>Your supervision commences on <span class="entry">{{ licence.actualReleaseDate | datetimeToDate or "MISSING - HDC Approved Date/Actual Release Date"}}</span> and expires on <span class="entry">{{ licence.licenceExpiryDate | datetimeToDate or "MISSING - Licence Expiry Date" }}</span> unless this licence is previously revoked.</li>
```
Comment (SebNegahbanMoJ) [inline server/views/pages/licence/HDC_AP.njk:63]: Is the start date of both of these meant to be ARD?

**Review by SebNegahbanMoJ — APPROVED** [review]

File: server/views/pages/licence/HDC_AP.njk (hunk 2 above)
Comment (louisenorris) [reply server/views/pages/licence/HDC_AP.njk:63]: Thanks Seb, that's a question we've added to a questions doc to double check and discuss at a later date 😅 so tbc but will be checked.

**Pull request merged**